website/docs/10_going-deeper/azure-scale.mdx

```diff
@@ -0,0 +1,42 @@
+import DocCardList from '@theme/DocCardList'
+
+# Scaling Booster Azure Functions
+
+Booster Azure Provider relies on CosmosDB [change feed processor](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/change-feed-processor) to consume new events.
+In CosmosDB, the partition keys are distributed in ranges, where each range represents a physical partition. [Unlike logical partitions,
+physical partitions are an internal implementation of the system and Azure Cosmos DB entirely manages physical partitions](https://learn.microsoft.com/en-us/azure/cosmos-db/partitioning-overview#physical-partitions)
+
+With Booster EventStream functionality, we could define the number of physical partitions our events are split and create instances for each partition.
+
+To enable EventStream, set the `EventStreamConfiguration` in the configuration object:
+
+> **Warning**: Currently, only available for Azure provider.
+
+```typescript
+  config.eventStreamConfiguration = {
+    enabled: true,
+    parameters: {
+      streamTopic: 'test',
+      partitionCount: 3,
+      messageRetention: 1,
+    },
+  }
+```
+
+## Parameters
+* StreamTopic: Define the internal topic name Booster will use.
+* PartitionCount: Number of Event Hub partitions. The number of functions app consumer instances will match the partition count
+* MessageRetention: Specifies the number of days to retain the events for this Event Hub
+
+## Infrastructure
+
+Enabling `EventStreamConfiguration` will apply some changes to the infrastructure:
+
+* Two functions will be created
+  * One function with a CosmosDB consumer that will produce Event Hubs events. Also, it will include the readModels functions, schedule functions app, etc...
+  * One function with an Event Hub consumer function app. This function will allow you to define the number of instances to be created
+* A new container to handle duplicated consumed events
+* A new Event Hub will be added for event handling.
```

Comment:
Aren't we missing "A Function App per partition"?

Reply:
There will be a Function App to consume the events with X instances. If the partition number is 3, there should be 3 instances of the `Event Hub consumer function app`